### PR TITLE
Add hw32 Concurrent

### PR DESCRIPTION
**Pull request by Leksandorr**

### CODE REVIEW

File: src/main/java/com/alevelhw/hw32/Task1.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            executorService.submit(runnable);
+        }
+
+        executorService.shutdown();
+
+        try {
+            executorService.awaitTermination(1, TimeUnit.HOURS);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```
Comment (Fabolo12) [inline src/main/java/com/alevelhw/hw32/Task1.java:32]: А это зачем?)

File: src/main/java/com/alevelhw/hw32/Task2.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+            watch.start();
+            do {
+                distance += Util.getRandomInt(100, 200);
+                try {
+                    Thread.sleep(Util.getRandomInt(400, 500));
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            } while (distance >= 1000);
```
Comment (Fabolo12) [inline src/main/java/com/alevelhw/hw32/Task2.java:68]: distance < 1000

**Review by Fabolo12 — APPROVED** [review]

File: src/main/java/com/alevelhw/hw32/Task2.java (hunk 2 above)
Comment (Leksandorr) [reply src/main/java/com/alevelhw/hw32/Task2.java:68]: упс

File: src/main/java/com/alevelhw/hw32/Task1.java (hunk 1 above)
Comment (Leksandorr) [reply src/main/java/com/alevelhw/hw32/Task1.java:32]: [shutdown](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutorService.html#shutdown--):

Initiates an orderly shutdown in which previously submitted tasks are executed, but no new tasks will be accepted. Invocation has no additional effect if already shut down.

This method does not wait for previously submitted tasks to complete execution. Use [awaitTermination](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutorService.html#awaitTermination-long-java.util.concurrent.TimeUnit-) to do that.

awaitTermination:

Blocks until all tasks have completed execution after a shutdown request, or the timeout occurs, or the current thread is interrupted, whichever happens first.

**Pull request merged**